express-api/src/controllers/ltsa/ltsaController.ts

```diff
@@ -1,6 +1,11 @@
 import { Request, Response } from 'express';
 import { stubResponse } from '@/utilities/stubResponse';
+import * as ltsaService from '@/services/ltsa/ltsaservice';
 
+export const getToken = async (req: Request, res: Response) => {
+  const token = await ltsaService.getTokenAsync();
+  return res.status(200).send(token);
+};
```

Comment:
No need to expose this as a controller. We don't want to send the token back to the frontend.
getTokenAsync() will be part of a controller that does all three stages of the LTSA request.